refresh/index.js

```diff
@@ -625,14 +641,6 @@ module.exports = generators.Base.extend({
       }
 
       this.fs.copyTpl(
-        this.templatePath('crud', 'GeneratedApplication.swift'),
-        this.destinationPath('Sources', 'Generated', 'GeneratedApplication.swift'),
-        { appName: this.projectName,
-          models: this.models,
-          metrics: this.metrics,
-          autoscale: this.autoscale }
-      );
-      this.fs.copyTpl(
         this.templatePath('crud', 'ApplicationConfiguration.swift'),
```

Comment:
NOTE: We should not copy `ApplicationConfiguration.swift` any more and should delete that file from `templates/crud`. The logic for config should be common to all the app types (albeit conditional on whether the app is cloud-enabled, as addressed in the line comment against the `basicweb` fragment).